App/tools/driver.makefile

```diff
@@ -166,8 +167,7 @@ export MAKE_FIRST
 export SUBMODULES
 export USE_LIBVERSION
 
-export ORIGIN=$(firstword $(shell git remote -v 2>/dev/null | awk '/psi.ch.*(fetch)/{print $$2;exit}')$(patsubst %,[%],$(shell git describe --tags --dirty --always --long 2>/dev/null)) $(addsuffix /,$(shell cat CVS/Root 2>/dev/null))$(shell cat CVS/Repository 2>/dev/null) $(PWD))
-
+export ORIGIN=$(firstword $(shell git remote -v 2>/dev/null | awk '/${GIT_DOMAIN}.*(fetch)/{print $$2;exit}')$(patsubst %,[%],$(shell git describe --tags --dirty --always --long 2>/dev/null)) $(addsuffix /,$(shell cat CVS/Root 2>/dev/null))$(shell cat CVS/Repository 2>/dev/null) $(PWD))
```

Comment:
Looks OK to me. Maybe I will extend it so allow multiple GIT_DOMAINs because people more and more tend to keep stuff in github.com/paulscherrerinstitute.

Reply:
That would be nice